project_task_stage_change_restriction/readme/CONFIGURE.md

```
@@ -0,0 +1,9 @@
+Go to "Project > Configuration > Task Stages" and select or create a new task stage.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)